test/powershell/engine/Formatting/ErrorView.Tests.ps1

```diff
@@ -61,12 +67,26 @@ Describe 'Tests for $ErrorView' -Tag CI {
 
         It "Pester Should shows test file and not pester" {
             $testScript = '1 + 1 | Should -Be 3'
-            $testScriptPath = Join-Path -Path $TestDrive -ChildPath 'mytest.ps1'
+
             Set-Content -Path $testScriptPath -Value $testScript
             $e = { & $testScriptPath } | Should -Throw -ErrorId 'PesterAssertionFailed' -PassThru
             $e | Out-String | Should -BeLike "*$testScriptPath*"
             $e | Out-String | Should -Not -BeLike '*pester*'
         }
+
+        It "Long lines should be rendered correctly" {
+            $testscript = @'
+                        $myerrors = [System.Collections.ArrayList]::new()
+                        Copy-Item (New-Guid) (New-Guid) -ErrorVariable +myerrors -ErrorAction SilentlyContinue
+                $error[0]
+'@
+
+            Set-Content -Path $testScriptPath -Value $testScript
+            $e = & $testScriptPath
+            $e | Out-String | Should -BeLike "*${testScriptPath}:2*"
+            # validate line number is shown
+            $e | Out-String | Should -BeLike '* 2 *'
```

Comment:
We do `$e | Out-String` twice - maybe use a variable?

Reply:
Good suggestion